src/plot_cartogram.R

```diff
@@ -282,3 +286,256 @@ rm_facet_clip <- function(svg_in, file_out, width){
   magick::image_write(img, file_out)
 
 }
+
+#' @description Adjusting legend for flow timeseries national plot - Instagram 
+#' @param plot_nat  Plot flow timeseries nationally
+#' @param enable_showtext enabiling fonts
+#' @param text_color Color of text in plot
+restyle_legend <- function(plot_nat, enable_showtext, text_color){
+  # import fonts
+  font_legend <- 'Noto Sans Mono'
+  font_add_google(font_legend)
+  showtext_opts(dpi = 300, regular.wt = 200, bold.wt = 700)
+  showtext_auto(enable = enable_showtext)
+  
+  # Restyle legend
+  plot_nat <- plot_nat +
+    guides(fill = guide_colorsteps(
+      title = "",
+      nrow = 1,
+      direction = 'horizontal',
+      label.position = "bottom",
+      barwidth = 12,
+      barheight = 0.6,
+      background = element_rect(fill = NA),
+      show.limits = TRUE,
+      even.steps = FALSE
+    )) +
+    theme(legend.background = element_rect(fill = NA),
+          text = element_text(family = font_legend, color = text_color, size = 6.5))
+}
```

Comment:
does it work to return the `get_legend(plot_nat)` here so the function is returning the styled legend rather than the whole plot?